src/Query/AST/Functions/Mysql/JsonPretty.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Scienta\DoctrineJsonFunctions\Query\AST\Functions\Mysql;
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Platforms\MySqlPlatform;
+use Doctrine\ORM\Query\AST\Functions\FunctionNode;
+use Doctrine\ORM\Query\Lexer;
+use Doctrine\ORM\Query\Parser;
+use Doctrine\ORM\Query\SqlWalker;
+
+/**
+ * "JSON_PRETTY" "(" string ")"
```

Comment:
Can you make sure the comment matches the parser function used.
In this way it is clear what the DQL parser does.
Putting `NewValue` in stead of `string` makes that much clearer.
I see now that this needs to be changed for `JSON_QUOTE` too (https://github.com/ScientaNL/DoctrineJsonFunctions/blob/763df144cc21eaaefe1ba3dbc771eab8c4e0c8d2/src/Query/AST/Functions/Mysql/JsonQuote.php#L13)